src/ethereum/ethereumTypes.ts

```diff
@@ -28,9 +28,9 @@ export interface EthereumInitOptions {
   blockchairApiKey?: string
   blockcypherApiKey?: string
   drpcApiKey?: string
-  /** For Etherscan v2 API */
+  /** @deprecated use evmScanApiKey instead */
   etherscanApiKey?: string | string[]
-  /** For bespoke scan APIs unsupported by Etherscan v2 API (e.g. fantomscan) */
+  /** For Etherscan v1 or v2 APIs */
```

Comment:
This comment change is egregiously wrong! We made a very explicit and intentional change in https://github.com/EdgeApp/edge-currency-accountbased/pull/924. 

This is also not in the scope of this PR.

Please revert this comment.